caffe2/CMakeLists.txt

```diff
@@ -1026,11 +1000,9 @@ endif()
 # ---[ CUDA library.
 if(USE_CUDA)
 
-  target_link_libraries(torch_cuda INTERFACE torch::cudart)
+  target_link_libraries(torch_cuda INTERFACE torch::cudart torch::nvtoolsext)
```

Comment:
torch::nvtoolsext should be a public dependency because the header is used in `torch_cuda`.
```
public = self + other
interface = other
private = self
```

Reply:
Oh just learned that, thanks